lib/src/view/tournament/tournament_screen.dart

```diff
@@ -0,0 +1,506 @@
+    final isLoggedIn = ref.watch(authSessionProvider)?.user.id != null;
+
+    if (verdicts.list.isEmpty) {
+      return const SizedBox.shrink();
+    }
+
+    return Row(
+      children: [
+        Icon(
+          isLoggedIn && verdicts.accepted ? Icons.check : Icons.lock,
+          color:
+              isLoggedIn
+                  ? verdicts.accepted
+                      ? LichessColors.good
+                      : LichessColors.error
+                  : null,
+          size: 30,
+        ),
+        const SizedBox(width: 10),
+        Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            for (final verdict in verdicts.list)
+              if (verdict.condition != 'Bot players are not allowed')
+                Text(
+                  verdict.condition,
+                  style: TextStyle(
+                    color:
+                        isLoggedIn
+                            ? verdict.ok
+                                ? LichessColors.good
+                                : LichessColors.error
+                            : null,
+                  ),
+                ),
+          ],
+        ),
+      ],
+    );
+  }
+}
+
+class _FeaturedGame extends ConsumerWidget {
+  const _FeaturedGame(this.featuredGame);
+
+  final FeaturedGame featuredGame;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return LayoutBuilder(
+      builder: (context, constraints) {
+        final boardSize = constraints.maxWidth;
+        switch (ref.watch(
+          tvControllerProvider(null, (featuredGame.id, featuredGame.orientation)),
+        )) {
+          case AsyncData(:final value):
+            {
+              final game = value.game;
+
+              final whitePlayer = _FeaturedGamePlayer(
+                game: game,
+                player: featuredGame.white,
+                side: Side.white,
+              );
+
+              final blackPlayer = _FeaturedGamePlayer(
+                game: game,
+                player: featuredGame.black,
+                side: Side.black,
+              );
+
+              return BoardThumbnail(
+                size: boardSize,
+                orientation: featuredGame.orientation,
+                fen: game.lastPosition.fen,
+                header: featuredGame.orientation == Side.white ? blackPlayer : whitePlayer,
+                footer: featuredGame.orientation == Side.white ? whitePlayer : blackPlayer,
+                lastMove: game.lastMove,
+              );
+            }
+          case _:
+            return BoardThumbnail(
+              size: boardSize,
+              fen: featuredGame.fen,
+              orientation: featuredGame.orientation,
+              header: const Shimmer(child: LoadingPlayerWidget()),
+              footer: const Shimmer(child: LoadingPlayerWidget()),
+            );
+        }
+      },
+    );
+  }
+}
+
+class _FeaturedGamePlayer extends StatelessWidget {
```

Comment:
I'd prefer the featured game board be like a broadcast mini board. Using the `header` and `footer` of `BoardThumbnail`.

Reply:
Can you elaborate? I'm already using `_FeaturedGamePlayer` as the `footer`/`header` of the `BoardThumbnail`

Reply:
I meant the style of the clock be like the ones in broadcasts (smaller clocks, orange when active).

Reply:
Done :)